### ISSUE TITLE
about  ADV_TYPE_DIRECT_IND_HIGH

### ISSUE THREAD

**Issue by IAMPIPP** [ECTO-1A/AppleJuice#54]
ESP32.If I use this type, I can't get any message in my phone.but if i use other types, I can't get any message after 0.5 meter.I am very sorry that I have a long time to use English.Hope you can understand what I said.Thanks.

**Comment by MaazSk** [comment]
I have created a PR to spam Android Devices can you check does it also have the same behaviour. Right now it only supports one device I will be adding support for more devices soon

**Comment by IAMPIPP** [comment]
> I have created a PR to spam Android Devices can you check does it also have the same behaviour. Right now it only supports one device I will be adding support for more devices soon

WoW，Can I see the code?Cause I have the same thought.But there is some reasons that I can't do this.So I need some experience to do this.

**Comment by MaazSk** [comment]
Yes, here is the link https://github.com/MaazSk/AppleJuiceEsp32/tree/feature/addRawData
This is the latest I have tried

**Comment by IAMPIPP** [comment]
> Yes, here is the link https://github.com/MaazSk/AppleJuiceEsp32/tree/feature/addRawData This is the latest I have tried This works on esp32 using esp-idf

Thank you.I need read that carefully.I can only understand python and arduino to this day.If you can use Arduino to make this project, that will be so cool.

**Comment by MaazSk** [comment]
Just give me some time will do that as it is easy to do in it I will make one more for it
